### ISSUE TITLE
Plesea help.

### ISSUE THREAD

**Issue by Alvazz** [naver/prismlivestudio#22]
LINK : fatal error LNK1104: can not open this file “C:\Users\Administrator\Documents\GitHub\prismlivestudio\build\windows\..\..\src\pris
m\libs\HMAC\lib\Release\x64\libApiGateway-MAC.lib” [C:\Users\Administrator\Documents\GitHub\prismlivestudio\src\prism\b
uild\main\frontend-api\frontend-api.vcxproj]

**Comment by zengvain** [comment]
We have submitted with https://github.com/naver/prismlivestudio/commit/5aa2dd8da2f0784a27f0d62722506633f6e09496 to resolve this issue, please try it again by pulling it. Thanks!

**Comment by Alvazz** [comment]
Thank you. I'll give it a try. If there are any questions, I'll get back to you. Thanks for sharing such a great project.

**Comment by Alvazz** [comment]
During compilation, many problems were encountered
![1](https://user-images.githubusercontent.com/50744096/153345423-a0de1a9e-387c-4043-9313-74bc262cee23.png)
I try to comment out the code and get another error
![2](https://user-images.githubusercontent.com/50744096/153345521-8fbb8d24-0361-4ad7-9e28-8b308f21913b.png)

![3](https://user-images.githubusercontent.com/50744096/153345539-a54bd721-e722-49a1-9a6e-2b08feb3373a.png)
![4](https://user-images.githubusercontent.com/50744096/153345545-e65bca07-f9f6-4f7f-b267-cf78a2e6984e.png)
So can I understand that the project is not as complete as open source? Or is it just for the GPL?

**Comment by Alvazz** [comment]
I use Qt version is "5.15.2"

**Comment by zengvain** [comment]
Yes, you should use Qt version **5.12.x**. It could not be built in 5.15.2. Sorry for that and I will change the description about it in hompage.
